src/main/java/training/subsystems/Arm/Wrist/WristConstants.java

```diff
@@ -0,0 +1,41 @@
+package training.subsystems.Arm.Wrist;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix.motorcontrol.can.TalonSRXConfiguration;
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public class WristConstants {
+    protected static final int WRIST_ID = 0;
```

Comment:
NO NEED WRIST BEFORE EVERY ONE
```suggestion
    protected static final int MOTOR_ID = 0;
```